robottelo/hosts.py

```diff
@@ -257,6 +257,24 @@ def subscription_manager_status(self):
     def subscription_manager_list(self):
         return self.execute('subscription-manager list')
 
+    def subscription_manager_get_pool(self, sub_list=[]):
+        pool_id = []
+        for sub in sub_list:
+            result = self.execute(
+                f'subscription-manager list --available --pool-only --matches="{sub}"'
+            )
+            result = result.stdout
+            result = result.split('\n')
+            result = ' '.join(result).split()
+            pool_id.append(result)
```

Comment:
`list.append(result.strip().split('\n'))` is this in one line works for you ? did you faced issue extra empty strings ?

Reply:
I initially tried that, but I was getting TypeError, I had to resort to the long format.